docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -189,12 +185,21 @@ The ticket request requires the token in one of the following formats:
 - Cookie named apimlAuthenticationToken.
 - Bearer authentication
 
-The request takes one parameter named `applicationName`, the name of the application for which the PassTicket should be generated. This parameter must be supplied.
+The request takes the `applicationName` parameter, which is the name of the application for which the PassTicket should be generated. This parameter must be supplied.
 
 The response is a JSON object, which contains information associated with the ticket.
 
 For more details, see the OpenAPI documentation of the API Mediation Layer in the API Catalog.
```

Comment:
This is the sentence I meant in my comment above about the proper API documentation of ZAAS. THis line should be placed before the description of the /login endpoint. And may be we should have the API doc somewhere for off-line access, otherwise we tell the user that they won't see it unless they install the whole ZOWE/APIML,